docs/data/material/components/snackbars/snackbars.md

```diff
@@ -18,8 +18,6 @@ They may contain a text action, but no icons. You can use them to display notifi
 
 {{"component": "modules/components/ComponentLinkHeader.js"}}
 
-**Frequency**: Only one snackbar may be displayed at a time.
-
 ## Simple snackbars
```

Comment:
Please don't use "simple". This notion differs from one individual to the other
```suggestion
## Basic snackbars
```

Now, since this is not the first h2, I assume this is not even what we want to convey as a message, maybe?

```suggestion
## Action buttons
```

Reply:
> Please don't use "simple". This notion differs from one individual to the other

This is the section in the single `<Snackbar />` component docs. I would not like to change it in this PR. 

Also why `Action buttons`? I didn't get it. It's a snackbar.

Reply:
Addressed in https://github.com/mui/material-ui/pull/31991/commits/b94cd195cdeee2fbb1e44bb4bb178c380d3e9a76 👍